packages/stream-metadata/src/environment.ts

```diff
@@ -79,6 +79,9 @@ function makeConfig() {
 		healthCheck: {
 			timeout: 5000, // 5 seconds
 		},
+		headers: {
+			invalidationId: 'X-StreamMetadata-Invalidation-Id',
+		},
```

Comment:
nit: @miguel-nascimento i think we should move this out to elsewhere because it's more of a constant and less of a config. it will never change. you can create a utils/consts.ts for this maybe?